node/resolvers/Mutations/Users.ts

```diff
@@ -509,6 +509,46 @@ const Users = {
       })
   },
 
+  createUserWithEmail: async (
+    _: void,
+    { orgId, costId, roleId, name, email, canImpersonate }: UserArgs,
+    ctx: Context
+  ) => {
+    const {
+      clients: { storefrontPermissions: storefrontPermissionsClient },
+      vtex: { logger },
+    } = ctx as any
+
+    try {
+      const result = await storefrontPermissionsClient.addUser({
+        orgId,
+        costId,
+        roleId,
+        name,
+        email,
+        canImpersonate,
+      })
+      return result.data.addUser
+    } catch (error) {
+      logger.error({
+        error,
+        message: 'addUser-error',
```

Comment:
This is the same message for the other `addUser` function, maybe better to have a different one here so we can distinguish them better, right?